public/docs/advanced/commands.md

```diff
@@ -132,55 +218,94 @@ Optional parameters:
 
 * `--clean`: Remove all network artifacts that aren't associated with a named network.
 
+
+### opcode
+
+Print the compiled opcodes for a given contract.
+
+```shell
+truffle opcode <contract_name>
+```
+
+Parameters:
+
+* `<contract_name>`: Name of the contract to print opcodes for. Must be a contract name, not a file name.
+
+
 ### publish
 
 Publish a package to the Ethereum Package Registry.
 
-```none
-$ truffle publish
+```shell
+truffle publish
 ```
 
-All parameters are pulled from your project's configuration file. See the [Package Management with EthPM](/docs/getting_started/packages-ethpm) section for more details.
+Takes no arguments. All parameters are pulled from your project's configuration file. See the [Package Management with EthPM](/docs/getting_started/packages-ethpm) section for more details.
+
 
 ### serve
 
-Serve the built app from `http://127.0.0.1:8080`, rebuilding and redeploying changes as needed. Like `truffle watch`, but with the web server component added.
+Serve the built app from `http://127.0.0.1:8080`, rebuilding and redeploying changes as needed. Similar to `truffle watch`, but with the web server component added.
 
-```none
-$ truffle serve
+```shell
+truffle serve [-p <port>] [--network <name>]
 ```
 
 Optional parameters:
 
-* `-p port`: Specify the port to serve on. Default is 8080.
-* `--network name`: Specify the network to use, using artifacts specific to that network.
+* `-p <port>`: Specify the port to serve on. Default is 8080.
+* `--network <name>`: Specify the network to use, using artifacts specific to that network. Network name must exist in the configuration.
+
 
 ### test
 
-Run all tests within the `./test` directory, or optionally run a single test.
+Run Mocha and Solidity tests.
```

Comment:
I'd replace Mocha with "Javascript" here, only because that's how we differentiate in other places. We should mention Mocha, but I believe that's listed on the Testing pages. Probably best to link to those pages somewhere under this command reference like you did with other ones.

Reply:
Sure. That line is what's listed in `truffle help`, so I went ahead and changed it there (and here) to "Run JavaScript and Solidity tests"